### PR TITLE
Static pages for new UI design: eligibility, FAQ, press. Plus social media sharing options in footer

### PR DESCRIPTION
**Pull request by mwellman17**
All updates made in /dev and do not impact existing views

Social Media share buttons in footer ([Trello](https://trello.com/c/oZSBDoK8/48-social-share-buttons-in-footer))
UI design update for Eligibility page
UI design update for FAQ page
Changed Home to Search in navbar per UI design prototype
Removed navbar link to Sites, per UI design prototype
Added Press page ([Trello](https://trello.com/c/LI8KjnLL/38-add-press-page-to-the-site-with-links))

Screen Shot 2021-02-14 at 8 56 24 PM
Screen Shot 2021-02-14 at 8 56 48 PM
Screen Shot 2021-02-14 at 8 58 07 PM
localhost_3002_dev_FAQ

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -1,4 +1,19 @@
 import Logo from './subcomponents/Logo';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import EmailLink from './subcomponents/EmailLink';
+import { faEnvelopeSquare } from "@fortawesome/free-solid-svg-icons";
+import { faTwitter, faFacebookSquare, faInstagram, faLinkedin } from "@fortawesome/free-brands-svg-icons";
+import { EmailShareButton, FacebookShareButton, LinkedinShareButton, TwitterShareButton, } from "react-share";
+
+/*
+   NOTE on social sharing: the react-share library includes support for the following fields:
```
Comment (hblumberg) [inline components/Footer.js:10]: Fantastic, thanks for including this!!

File: components/NavHeader.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const NavHeader = () => {
   const router = useRouter()
 
   const renderLinkClass = (nav) => {
-    const path = nav === "home" ? "/" : nav;
+    const path = nav === "home" ? "/dev" : nav;
```
Comment (hblumberg) [inline components/NavHeader.js:13]: Maybe worth dropping a TODO so we remember to fix this when "launching"?

File: components/Footer.js [hunk 3]
```diff
@@ -1,4 +1,19 @@
 import Logo from './subcomponents/Logo';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import EmailLink from './subcomponents/EmailLink';
+import { faEnvelopeSquare } from "@fortawesome/free-solid-svg-icons";
+import { faTwitter, faFacebookSquare, faInstagram, faLinkedin } from "@fortawesome/free-brands-svg-icons";
+import { EmailShareButton, FacebookShareButton, LinkedinShareButton, TwitterShareButton, } from "react-share";
```
Comment (hblumberg) [inline components/Footer.js:4]: Nit, no pressure to change: I usually find it helpful to separate third party and local dependencies. So something like:

```
import { EmailShareButton, FacebookShareButton, LinkedinShareButton, TwitterShareButton, } from "react-share";
import { faEnvelopeSquare } from "@fortawesome/free-solid-svg-icons";
import { faTwitter, faFacebookSquare, faInstagram, faLinkedin } from "@fortawesome/free-brands-svg-icons";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";

import EmailLink from './subcomponents/EmailLink';
import Logo from './subcomponents/Logo';
```

File: pages/dev/FAQ.js [hunk 4]
```diff
@@ -1,9 +1,34 @@
-import Layout from "../../components/Layout"
+import Layout from "../../components/Layout";
+import EmailLink from '../../components/subcomponents/EmailLink';
```
Comment (hblumberg) [inline pages/dev/FAQ.js:2]: Love the reuse!

File: pages/dev/eligibility.js [hunk 5]
```diff
@@ -1,9 +1,26 @@
 import Layout from "../../components/Layout"
 
+// TODO: should the phase and group be set to variables?
```
Comment (hblumberg) [inline pages/dev/eligibility.js:3]: 😍

File: pages/dev/press.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+      </a>
+      <h2>Framingham Source</h2>
+      <a href="https://framinghamsource.com/index.php/2021/02/10/framingham-high-graduates-helping-people-book-covid-vaccines-through-vaccinatema-com-website/" target="_blank">
+        Framingham High Graduates Helping People Book COVID Vaccines Through VaccinateMA.com Website
+      </a>
+    </div>
+  </Layout>
+);
+
+export default Press;
```
Comment (hblumberg) [inline pages/dev/press.js:53]: Would be cool to eventually pull these from a database, but this definitely works for now!

**Review by hblumberg — APPROVED** [review]
This is looking so great!! We'll definitely want Harlan's eyes on this at some point, but I think doing design review separately makes sense.

And thanks for the screenshots! To confirm, no console errors or anything?

**Comment by mwellman17** [comment]
> To confirm, no console errors or anything?

Console is clean

File: pages/dev/press.js (hunk 6 above)
Comment (mwellman17) [reply pages/dev/press.js:53]: Refactoring to use tile components, so adding press pieces or hooking up to an endpoint will be easier in the future

**Comment by johndeyrup** [comment]
looks goood

**Review by johndeyrup — APPROVED** [review]

**Pull request merged**